agreements/project-contribution/asset-transfer-agreement.md

```diff
@@ -71,11 +71,11 @@ The attached [schedules of assets](#schedules) identify the assets being transfe
 
 ## Transfer of Assets
 
-Assignor hereby transfers, and the Sponsor hereby acquires, the Assets identified in the schedules below and all rights in those Assets, including all intellectual property rights.
+Assignor hereby transfers to the Sponsor all right, title, and interest in and to the Assets identified in the schedules below, including any intellectual property rights in those Assets that are owned by Assignor.
```

Comment:
Is the first occurrence of "right" intentionally singular, or should it be plural?